examples/src/content/jcr_root/content/dam/core-components-examples/library/aem-corecomponents-logo.svg/.content.xml

```diff
@@ -13,6 +13,7 @@
             dam:sha1="7d16c64c03a8f051d238f2e4e26edef54fb4a636"
             dam:size="{Long}18575"
             dc:format="image/svg+xml"
+            dc:title="Component Library"
```

Comment:
Could we call this one "AEM Core Components Logo"

Reply:
@richardhand not sure about this one @gabrielwalt may have an opinion on this - it's the alt text for the logo at the top left corner

Reply:
@reusr1 - Shouldn't the alt represent the content of the image? Here this is a general logo for core components non-specific to the library. If re-used in another context it would no longer be semantic.

If we want do want to rename it, I would recommend at the component level - but I'm not sure that's necessary (although I'm no SEO expert).

Would a description at the component level help us here as it presents a title attribute 
- "Component Library Home" or similar.

Reply:
I agree with you - it's a problem of the teaser and the use of the image in the teaser. The alt text for the image in a teaser should be driven by the teaser and passed to the image. I would create a separate issue for this. Btw: this is for accessibility (screen readers for example) and not SEO.

Reply:
This logo image isn't used within a teaser component. 

> The alt text for the image in a teaser should be driven by the teaser
you mean here the ability in the teaser dialog to influence the standard image properties (title and description)? 

Isn't alt often fairly constant - I mean if an image content is correctly described in the asset it's unlikely you would want to change that - or do you see it often shifting based on context? On a side note, it's strange to me that alt is derived from the title and title from the description.

> this is for accessibility (screen readers for example) and not SEO.
was a late night misspeak :P

Reply:
same here - the logo is the only one that's an image, so the issue with what I see with the teaser does not apply on it. I changed the dc:title to match the text in the image (that makes most sense for an alt text).